src/app/product-overview/product-overview.component.html

```diff
@@ -0,0 +1,8 @@
+<h1>Product overview</h1>
+
+<div>
+    <div class="product" *ngFor="let product of products">
+        <h3>{{ product.name }}</h3>
+        <p>&euro;{{ product.price }}</p>
```

Comment:
Could we change this to a dollar sign? The seems supplier to always provide them in their local currency(dollar).